plugins/webp-uploads/settings.php

```diff
@@ -111,8 +132,9 @@ function webp_uploads_generate_avif_webp_setting_callback(): void {
  * @since 1.0.0
  */
 function webp_uploads_generate_webp_jpeg_setting_callback(): void {
-	?>
 
+	?>
+	<tr><td colspan="2" class="td-full">
```

Comment:
I see now why the month-based organization checkbox field is so narrow.

`trunk` | This branch
--|--
![image](https://github.com/WordPress/performance/assets/134745/292203cc-2a1c-4a8c-abb1-05554f450ca0) | ![image](https://github.com/WordPress/performance/assets/134745/7ae1f35e-5358-4f1a-a659-847deb33c42b)

 It's because core is expecting the entire table to have only one column. So when the plugin adds additional rows which have two columns, then the result is the core-supplied cell is constrained to the first column.

I wonder why core is using a table here in the first place then? 😕

Reply:
i'm removing the two column layout for now

Reply:
Oh, on multisite it _does_ have two columns 🤦‍♂️ 

https://github.com/WordPress/wordpress-develop/blob/415995a2101e6b4546931eb29e3eca08c3b76e13/src/wp-admin/options-media.php#L156-L160

Reply:
oof. what happens on multisite when we leave off the columns?